cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/kyma-project/cli/internal/clusterinfo"
 	"github.com/kyma-project/cli/internal/kube"
 	"github.com/kyma-project/cli/pkg/asyncui"
-
+	"github.com/magiconair/properties"
```

Comment:
@clebs :; not sure if it's worth to use a properties-parser for validating hte CLI parameters. Maybe better to extract key/value pairs by our own. What do you think?

Reply:
Sounds good to me, it saves us some parsing and might be more helpful in the future, since it can recursively parse and supports struct tags.
Also a good point is, this package has no dependencies on its go.mod file (otherwise I would have advised against it).

Reply:
👍